applications/adv_networking_bench/adv_networking_bench_default_tx_rx.yaml

```diff
@@ -95,4 +96,3 @@ bench_tx:
   ip_dst_addr: <5.6.7.8>  # Destination IP address - required on layer 3 network
   udp_src_port: 4096      # UDP source port
   udp_dst_port: 4096      # UDP destination port
-  address: <0000:00:00.0> # Source NIC Bus ID. Should match the address of the Tx interface above
```

Comment:
Still need to update all other configs.

Also need to update the `##### Configure the application` section in the tutorial.

I'm happy to do it.

Reply:
Sure, if you can that would be helpful.

Reply:
if you wanted to review: https://github.com/nvidia-holoscan/holohub/compare/1cb11f265ddfbaf65376e830dad399c31b8cb15b..fc4603c739fde3a1c9faddae15d12c55bb7983b5